renderdoc/driver/metal/metal_device.h

```diff
@@ -80,7 +94,12 @@ class WrappedMTLDevice : public WrappedMTLObject
     TypeEnum = eResDevice
   };
 
+  static uint64_t nextDrawableTLSSlot;
+  static IMP real_CAMetalLayer_nextDrawable;
```

Comment:
Not sure how many ObjC methods will be swizzled in the end - if there are more then will move them to a helper struct with helper methods and out of WrappedMTLDevice